typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -637,8 +637,8 @@ export const solana: ChainMetadata = {
 
 export const solanatestnet: ChainMetadata = {
   protocol: ProtocolType.Sealevel,
-  chainId: 13998111450,
-  domainId: 13998111450,
+  chainId: 1399811150,
+  domainId: 1399811150,
```

Comment:
Typo in the testnet id, sorry! Not in use yet, hope folks don't mind if I throw this fix in here